backend/tests/v5/scripts/teamspaces/createTeamspace.test.js

```diff
@@ -23,19 +23,45 @@ const {
 	generateUserCredentials,
 } = require('../../helper/services');
 
-const { src, utilScripts } = require('../../helper/path');
+const { createAccount } = require('../../helper/fronteggMock');
+
+const { src, srcV4, utilScripts } = require('../../helper/path');
 
 const CreateTeamspace = require(`${utilScripts}/teamspaces/createTeamspace`);
 
 const { disconnect } = require(`${src}/handler/db`);
-const { templates } = require(`${src}/utils/responseCodes`);
+
+jest.mock('../../../../src/v4/models/userBilling');
+const UserBillingMock = require(`${srcV4}/models/userBilling`);
 
 const user = generateUserCredentials();
+const emailUser = generateUserCredentials();
+emailUser.user = 'emailUser';
+emailUser.basicData.email = 'test@email.com';
 const teamspace = generateRandomString();
+const existingAccount = generateRandomString();
+let existingAccountId;
+
+jest.mock('../../../../src/v5/models/users', () => {
```

Comment:
Script tests are integration tests, so we don't want to mock a function unless it's absolutely necessary (e.g. it's talking to an external service/ checking for an email etc)

We should create the users using the service helper as necessary.